aderyn_core/src/detect/test_utils/load_source_unit.rs

```diff
@@ -5,13 +5,27 @@ use std::{
     sync::Arc,
 };
 
-use crate::visitor::ast_visitor::Node;
-use crate::{ast::SourceUnit, context::workspace_context::WorkspaceContext};
+use crate::{
+    ast::SourceUnit,
+    context::{graph::traits::Transpose, workspace_context::WorkspaceContext},
+};
+use crate::{context::graph::WorkspaceCallGraph, visitor::ast_visitor::Node};
 
 use super::ensure_valid_solidity_file;
 
+#[cfg(test)]
+pub fn load_solidity_source_unit_with_callgraphs(filepath: &str) -> WorkspaceContext {
```

Comment:
As a follow up - we should probably just remove this option so that callgraphs are loaded for every test harness. More choices/options are just more DevRel questions that we can solve by including all the bells and whistles in one load_solidity_source function.